test/service/test_manifest.py

```diff
@@ -1845,6 +1845,12 @@ def test_compact_manifest(self):
                 '',
                 ''
             ),
+            (
+                'datasets.duos_id',
```

Comment:
This test is indexing the DUOS bundle so I would expect the DUOS ID to be included in the resulting manifest. Please find out why it's not.

Reply:
The DUOS ID value is not being included in the manifest since the DUOS bundle is only added to the `datasets` and `bundles` indices, and not the `files` index which the manifest pulls values from.